discord/interactions.py

```diff
@@ -111,18 +111,20 @@ def _from_data(self, data: InteractionPayload):
         except KeyError:
             self.message = None
 
-        try:
-            self.user = User(state=self._state, data=data['user'])
-        except KeyError:
-            self.user = None
-
+        self.user = None
         # TODO: there's a potential data loss here
         if self.guild_id:
             guild = self.guild or Object(id=self.guild_id)
             try:
                 self.user = Member(state=self._state, guild=guild, data=data['member'])
             except KeyError:
                 pass
```

Comment:
What happens here then?

Reply:
Moved it outside the conditional to avoid putting it twice in `except KeyError`
Which was previously being set to None as well